presto-main/src/main/java/com/facebook/presto/operator/scalar/VarbinaryFunctions.java

```diff
@@ -315,6 +315,38 @@ public static long crc32(@SqlType(StandardTypes.VARBINARY) Slice slice)
         return crc32.getValue();
     }
 
+    @Description("compute fnv-1 32 bit")
+    @ScalarFunction("fnv1_32")
+    @SqlType(StandardTypes.BIGINT)
+    public static long fnv1Hash32(@SqlType(StandardTypes.VARBINARY) Slice slice)
+    {
+        return FnvHash.fnv1Hash32(slice);
```

Comment:
Please static import the functions per coding convention.

Reply:
So right now, the method names in VarbinaryFunctions are the same as method names in FnvHash. I think if we use static imports, by default, the code will call the locally defined functions, which will cause stack overflow due to the recursive call.

Should I rename the functions to be fnv1Hash32VarBinary(), etc?

Reply:
Oh didn't realize that. This is fine then. Thanks!